### ISSUE TITLE
Treeplugin startup with error "first argument of unbound method must have type 'ColourDatabase'"

### ISSUE THREAD

**Issue by hewittex16** [robotframework/RIDE#2628]
1. Startup robotide with command `python -m robotide.__init__`
2. Error raised and the window don't show.
3. Downgrade to `2.0.6`, it works.
4.  For `2.0.7`,   I fix this by modify this line in `robotide/ui/treeplugin.py`
    ```
    self.SetItemTextColour(node, wx.ColourDatabase.Find(colourName="GRAY"))
    ```
    to
    ```
    self.SetItemTextColour(node, wx.TheColourDatabase.Find(colourName="GRAY"))
    ```
5. Full log
 ```
$ python -m robotide.__init__                                                                         
02:16:33 PM: Debug: ClientToScreen cannot work when toplevel window is not shown
02:16:33 PM: Debug: ClientToScreen cannot work when toplevel window is not shown
Error in listener: Tree._resource_added_1600, topic: ride.open.resource

Traceback (most recent call last):
None
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/pubsub/core/topicobj.py", line 482, in __sendMessage
    listener(data, self, allData)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/pubsub/core/listener.py", line 237, in __call__
    cb(**kwargs)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 480, in _resource_added
    parent = self._get_dir_node(ctrl.parent)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 490, in _get_dir_node
    parent = self._get_dir_node(ctrl.parent)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 490, in _get_dir_node
    parent = self._get_dir_node(ctrl.parent)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 491, in _get_dir_node
    self._render_datafile(parent, ctrl)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 526, in _render_datafile
    self._render_datafile(node, child)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 517, in _render_datafile
    node = self._create_node_with_handler(parent_node, controller, index)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 564, in _create_node_with_handler
    self._set_item_excluded(node)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 341, in _set_item_excluded
    self.SetItemTextColour(node, wx.ColourDatabase.Find("GRAY"))
Traceback (most recent call last):
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/application/application.py", line 121, in OnInit
    self.treeplugin.populate(self.model)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 141, in populate
    self._tree.populate(self._model)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 447, in populate
    self._populate_model(model)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 470, in _populate_model
    self._render_datafile(self.root, model.data, 0)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 526, in _render_datafile
    self._render_datafile(node, child)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 517, in _render_datafile
    node = self._create_node_with_handler(parent_node, controller, index)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 564, in _create_node_with_handler
    self._set_item_excluded(node)
  File "/home/user/data/python_venv/myproject/lib/python3.10/site-packages/robotide/ui/treeplugin.py", line 341, in _set_item_excluded
    self.SetItemTextColour(node, wx.ColourDatabase.Find("GRAY"))
TypeError: ColourDatabase.Find(): first argument of unbound method must have type 'ColourDatabase'
OnInit returned false, exiting...
Exception ignored in atexit callback: <built-in function _wxPyCleanup>
wx._core.wxAssertionError: C++ assertion "GetEventHandler() == this" failed at /tmp/pip-install-b9bkpapg/wxpython_b27059f9a75f4598aa24fe265dc7954e/ext/wxWidgets/src/common/wincmn.cpp(473) in ~wxWindowBase(): any pushed event handlers must have been removed
```

**Comment by HelioGuilherme66** [comment]
@hewittex16 Please indicate what Linux system are you using and also the version of wxPython. Thanks.

**Comment by hewittex16** [comment]
Ubuntu 22.04 
wxPython 4.2.1

**Comment by HelioGuilherme66** [comment]
This bug is only reproducible, when having excluded files or directories.

**Comment by HelioGuilherme66** [comment]
@hewittex16 Thanks for the report and fix. This is now fixed in v2.0.8dev9.
